shell/client/admin-client.js

```diff
@@ -760,13 +790,11 @@ Template.adminAdvanced.events({
     const state = Iron.controller().state;
     const token = this.token;
     resetResult(state);
-    state.set("numSettings", 12);
+    state.set("numSettings", 11);
```

Comment:
The number is the number of settings that we're saving, which is reduced by one because `returnAddress` is no longer present in this form.

I agree it's weird, and I'll try to make saving settings be an atomic action per-page in the new UI, rather than one with 11 independently succeeding-or-failing parts, where possible :P